webapp/src/components/post_type_jitsi/post_type_jitsi.tsx

```diff
@@ -61,6 +60,7 @@ export class PostTypeJitsi extends React.PureComponent<Props, State> {
                 const props = this.props.post.props;
                 let meetingLink = props.meeting_link + '?jwt=' + (this.state.meetingJwt);
                 meetingLink += `#config.callDisplayName="${props.meeting_topic || props.default_meeting_topic}"`;
+                meetingLink = encodeURI(meetingLink);
```

Comment:
I'm thinking we should escape each part of the link using `encodeURIComponent`, instead using `encodeURI`